### DISCUSSION TITLE
Question about behavior: virtual-list

### DISCUSSION THREAD

**Discussion by 000o0** [c-smile/sciter-js-sdk#59] [Q&A]
How to insert / delete virtual-list items dynamically?

**Comment by c-smile** [comment]
virtual list is an ordinary DOM element. It contains N DOM elements that reflect fragment of recordset.

So this: 
```
vlist.append(<div key={index} mine="" >
        <div.text state-html={item.html} />
        <img src={ item.avatar } />        
      </div>);
```
will add the element as a child. After that you can do ```vlist.lastElementChild.scrollIntoView()``` to make it visible.
